pype/lib.py

```diff
@@ -469,6 +469,40 @@ def get_version_from_path(file):
         )
 
 
+def get_last_version_from_path(path_dir, filter=None):
```

Comment:
Does this method do what the name says if filter is not entered?

Reply:
well, I believe so. Didn't it work for you?

Reply:
Nope, in maya (for example) it returns `workspace.mel`. If anybody has other files in workfiles folder it may return theirs files instead of workfile.

Reply:
and what is your suggestion then?

Reply:
> Nope, in maya (for example) it returns `workspace.mel`
I believe it might be due unset filter arg. In that case the `filter` attribute has to be flagged as required instead of optional. I will add this into docstring with example.